runtime/tests/integration_tests.rs

```diff
@@ -31,6 +32,35 @@ fn run_rt_test(test_bin_name: Option<&str>) {
     .unwrap();
 
     model.upload_firmware(&image).unwrap();
+
+    model
+}
+
+fn send_mailbox_command(model: &mut DefaultHwModel, cmd_id: u32, arg_buf: &[u32]) -> Vec<u32> {
+    model.soc_mbox().lock().read().lock();
+    model.soc_mbox().cmd().write(|_| cmd_id);
+    model.soc_mbox().dlen().write(|_| arg_buf.len().try_into().unwrap());
+
+    for word in arg_buf {
+        model.soc_mbox().datain().write(|_| *word);
+    }
+    model.soc_mbox().execute().write(|exec_reg| exec_reg.execute(true));
+
+    model.step_until(|m| {
+        m.soc_mbox().status().read().status().cmd_complete()
```

Comment:
Should be:
model.step_until(|m| {
        !m.soc_mbox().status().read().status().cmd_busy()).